labels.json

```diff
@@ -0,0 +1,119 @@
+[
+  {
+    "name": "addition/proposal",
+    "description": "New features or enhancements",
+    "color": "d4c5f9"
+  },
+  {
+    "name": "anchor permanence",
+    "description": "Identifies issues opened by other standards organizations, as per WHATWG's [anchor permanence policy](https://whatwg.org/working-mode#anchors)",
+    "color": "ececec",
+    "url_exclude_is_open": true
+  },
+  {
+    "name": "clarification",
+    "description": "Standard could be clearer",
+    "color": "fef2c0"
+  },
+  {
+    "name": "compat",
+    "description": "Standard is not web compatible",
+    "color": "bfd4f2"
+  },
+  {
+    "name": "do not merge yet",
+    "description": "Pull request must not be merged per rationale in comment",
+    "color": "e11d21"
+  },
+  {
+    "name": "good first issue",
+    "description": "Ideal for someone new to a WHATWG standard or software project",
+    "color": "bfe5bf"
+  },
+  {
+    "name": "i18n-alreq",
+    "description": "Used by the internationalization community to notify Arabic script experts of relevant issues",
+    "color": "d4c5f9"
```

Comment:
I think it'd be nice to use something different than the addition/proposal color